lib/transfer.cpp

```diff
@@ -123,6 +136,16 @@ namespace quda {
     param.fieldOrder = location == QUDA_CUDA_FIELD_LOCATION ? QUDA_FLOAT2_FIELD_ORDER : QUDA_SPACE_SPIN_COLOR_FIELD_ORDER;
     param.setPrecision(location == QUDA_CUDA_FIELD_LOCATION ? null_precision : B[0]->Precision());
 
+    if (is_staggered) { 
```

Comment:
@maddyscientist re: assigning pointers to `std::numeric_limits<long long unsigned int>::max()`

Reply:
change to `uint64_t`

Reply:
Did a sweep with `grep`, replaced it everywhere